tests/foreman/ui/test_org.py

```diff
@@ -852,23 +871,23 @@ def test_add_configtemplate(self, template_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        entities.ConfigTemplate(name=template_name).create()
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_templates=[template_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_template'])
-            element = session.nav.wait_until_element(
-                (strategy, value % template_name))
-            self.assertIsNotNone(element)
+            for template_name in generate_strings_list():
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence as well as increase of entities associated to the organization. Not a blocker for this PR.